packages/app/src/main.ts

```diff
@@ -5,10 +5,12 @@ import urql from '@urql/vue'
 import App from './App.vue'
 import { makeUrqlClient } from '@packages/frontend-shared/src/graphql/urqlClient'
 import { createI18n } from './locales/i18n'
+import { createRouter } from './router/router'
```

Comment:
Can we follow the barreling convention and create `index.ts` inside of `router` that exports the router instance (and whatever else).

Reply:
IIRC we don't like directory `index.ts` files because it's a pain to search for. I'd take a `router.ts` at the sibling level, but I wasn't sure what else we'd need, so I made a router directory. `locales/i18n.ts` is currently mimicking this pattern, too.